tests/dhcp_server/dhcp_server_test_common.py

```diff
@@ -86,17 +86,18 @@ def apply_dhcp_server_config_cli(duthost, config_commands):
         duthost.shell(cmd)
 
 
-def apply_dhcp_server_config_gcu(duthost, config_to_apply):
+def apply_dhcp_server_config_gcu(duthost, config_to_apply, ignore_errors=False):
     logging.info("The dhcp_server_config: %s" % config_to_apply)
     tmpfile = duthost.shell('mktemp')['stdout']
     try:
         duthost.copy(content=json.dumps(config_to_apply, indent=4), dest=tmpfile)
         output = duthost.shell('config apply-patch {}'.format(tmpfile), module_ignore_errors=True)
-        pytest_assert(not output['rc'], "Command is not running successfully")
-        pytest_assert(
-            "Patch applied successfully" in output['stdout'],
-            "Please check if json file is validate"
-        )
+        if not ignore_errors:
```

Comment:
Do we have scenario that can ignore error?

Reply:
This change is to avoid the error described in: https://github.com/sonic-net/sonic-mgmt/blob/0e5793c3e360c723dc1447a7c7cbe748098ec3e1/tests/configlet/util/generic_patch.py#L157C1-L161C57

Reply:
By the way, it's an optional parameter, it doesn't break any logic and it depends on the user's choice.